docs/source/example_data_types.ipynb

```diff
@@ -0,0 +1,404 @@
+    "\n",
+    "For some additional insight, you are welcome to \n",
+    "[look at this post](https://vaex.io/blog/a-hybrid-apache-arrow-numpy-dataframe-with-vaex-version-4) \n",
+    "as well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Supported Data Types in Vaex\n",
+    "\n",
+    "In the table below we define:\n",
+    "\n",
+    " - Supported: a column or expression of that type can exist and can be stored in at least one file format;\n",
+    " - Unsupported: a column or expression of that type can currently not live within a Vaex DataFrame, but can supported be added in the future;\n",
+    " - Will not support: This datatype will not be supported in Vaex going forward. \n",
+    " \n",
+    "\n",
+    "\n",
+    "| Framework       | Dtype           | Supported     | Remarks                                       |\n",
+    "|-----------      |--------------   |-----------    |---------------------------------------------  |\n",
+    "| Python          | `int`           | yes           | Will be converted to a `numpy` array          |\n",
+    "| Python          | `float`         | yes           | Will be converted to a `numpy` array          |\n",
+    "| Python          | `datetime`      | not yet       |                                               |\n",
+    "| Python          | `timedelta`     | not yet       |                                               |\n",
+    "| Python          | `str`           | yes           | Will be converted to Arrow array              |\n",
+    "| `numpy`         | `int8`          | yes           |                                               |\n",
+    "| `numpy`         | `int16`         | yes           |                                               |\n",
+    "| `numpy`         | `int32`         | yes           |                                               |\n",
+    "| `numpy`         | `int64`         | yes           |                                               |\n",
+    "| `numpy`         | `float16`       | yes           | Operations should be upcast to `float64`      |\n",
+    "| `numpy`         | `float32`       | yes           |                                               |\n",
+    "| `numpy`         | `float64`       | yes           |                                               |\n",
+    "| `numpy`         | `datetime64`    | yes           |                                               |\n",
+    "| `numpy`         | `timedelta64`   | yes           |                                               |\n",
+    "| `numpy`         | `object ('O')`  | no            |                                               |\n",
```

Comment:
should this be partial? because the array can live inside the dataframe, you just cannot export it.